src/ui/layer_list_panel.css

```diff
@@ -11,9 +11,24 @@
 .neuroglancer-layer-list-panel-item {
   display: flex;
   flex-direction: row;
-  padding: 2px;
+  padding: 2px 4px;
   border: 1px solid #aaa;
   margin: 2px;
+  align-items: center;
+  gap: 4px;
+}
+
+.neuroglancer-layer-list-panel-item input[type="checkbox"] {
+  width: 1rem;
+  height: 1rem;
+  margin: 4px;
+}
+
+.neuroglancer-layer-list-panel-item .neuroglancer-icon {
```

Comment:
Is this targeting just the eye icon? If so it would be better to give that a class to make this more explicit